src/routes/+page.svelte

```diff
@@ -25,10 +25,16 @@
 		{ max: 5 },
 	);
 
-	const restart_log = task.restart(async (param: number) => {
-		await new Promise((r) => setTimeout(r, 2000));
-		return param;
-	});
+	const restart_log = task.restart(
+		async function* (param: number) {
+			console.log('started ', param);
+			await new Promise((r) => setTimeout(r, 2000));
+			yield;
+			console.log('finished ', param);
```

Comment:
I think we can either remove the logs or make it clearer that they are for the restartable task.

Also, do we need the `yield` here?